R/read_vpts.R

```diff
@@ -1,241 +1,129 @@
-#' Read a time series of vertical profiles (`vpts`) from file
-#'
-#' @param file A text file containing the standard output (stdout) generated
-#' by vol2bird (or the package function `calculate_vp`).
-#' @param lon numeric. Longitude of the radar in decimal degrees.
-#' @param lat numeric. Latitude of the radar in decimal degrees.
-#' @param height numeric. Height above sea level of the radar antenna in meters.
-#' @param radar A string containing a radar identifier.
-#' @param wavelength Radar wavelength in cm, or one of 'C' or 'S' for C-band
-#' and S-band radar, respectively, in which case C-band wavelength is assumed
-#' to be 5.3 cm and S-band wavelength 10.6 cm
-#' @param sep the field separator character, see [utils::read.table]
-#'
-#' @return An object inheriting from class `vpts`, see
-#' [`vpts()`][summary.vpts] for details.
+#' Read time series of vertical profiles (`vpts`) from file(s)
 #'
+#' Reads `vpts` data from one or more files.
+#' The following file formats are supported (but cannot be mixed):
+#' - [VPTS CSV](https://aloftdata.eu/vpts-csv/).
+#' - [ODIM bird profile](https://github.com/adokter/vol2bird/wiki/ODIM-bird-profile-format-specification).
+#' - vol2bird standard output (see example below).
```

Comment:
Getting a conflicting warning message:

```
> stdout_file <- system.file("extdata", "example_vpts.txt", package = "bioRad")
> 
> read_vpts(stdout_file)
No extension detected; assuming file type .txt which maps to stdout format
Error in (function (file, radar, lat, lon, height, wavelength = "C", sep = "")  : 
  'radar' argument missing. Required to specify a radar identifier.
In addition: Warning message:
In read_vpts(stdout_file) :
  .txt extenstion detected - falling back to read_stdout().

    Please consider updating your workflow by using VPTS csv or h5 input files
 ```

Reply:
this file does have a .txt extension, but function says no extension detected

Reply:
fixed in [92a8556](https://github.com/adokter/bioRad/pull/590/commits/92a855666929e386539416095a73ca2dab48068c)